frontend/src/components/UserSearch.vue

```diff
@@ -23,7 +23,7 @@
           dense
           v-model="userSearchInput"
           placeholder="Username, email, name, or ID"
-          @keyup.native.enter="searchUser"
+          @keyup.enter="searchUser('&search='+userSearchInput)"
```

Comment:
This fixes a bug where the Basic Search was failing when you press "Enter" in the text field. The method call isn't new, it's actually the same method call when you click the Search button:
```html
<v-btn id="search-button" class="secondary" medium @click.native="searchUser('&search='+userSearchInput)">
```

I think Gus just missed updating it when he was adding role search, or something.